torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -0,0 +1,130 @@
+import enum
+from typing import Tuple
+
+import torch
+import torch.testing._internal.dist_utils as dist_utils
```

Comment:
Can we test that `forward` and `forward_async` can be called within a torchscript function?

Reply:
Adding.